### DISCUSSION TITLE
Separatist uniforms are hard to tell apart

### DISCUSSION THREAD

**Discussion by ubertpendragon** [FakeFishGames/Barotrauma#14500] [Unstable feedback]
New separatist uniforms lack contrast/saturation so they're pretty hard to tell apart and turn into a brown/green mush
I know their visual identity is "dirty" but it's like they're covered in mud at times

Full brightness comparison (the high-visibility engineer vests are almost low-visibility)
![image](https://github.com/user-attachments/assets/cb1355c3-2999-4039-90be-5402439425c4)

Dimmed light environment
![image](https://github.com/user-attachments/assets/eccc9e6a-5bd9-415c-b820-3af103ca1814)

Camo pattern is almost invisible
![image](https://github.com/user-attachments/assets/8b6972e3-fe00-4f5d-b5b2-2f53df6b485d)
Increased contrast/saturation
![image](https://github.com/user-attachments/assets/363d14c9-4f63-4732-be6b-d8f8c53dda67)

**Comment by Vykn** [comment]
An opinion on the recent sprite changes:
![image](https://github.com/user-attachments/assets/9faee915-0ff6-4a86-8dab-0a18408baefb)

Medic uniform 1 is now just a slightly greener version of the regular medic's fatigues. The tiger camo was really unique and the cross on the arm already makes it extremely distinct as a medic uniform. I think it would be a shame to replace it.

**Reply by ubertpendragon** [reply]
"Tiger" camo looked too much like other brown uniforms


I still think the engineer vests could be brigher, especially engineer 1 he looks like a regular soldier
